package.json

```diff
@@ -50,7 +50,7 @@
     "fix:license": "gulp update-license-headers",
     "clean": "rm -rf .go_workspace .tmp coverage dist npm-debug.log",
     "postversion": "node aio/scripts/version.js",
-    "postinstall": "node aio/scripts/version.js && ./node_modules/protractor/bin/webdriver-manager update --gecko=false"
+    "postinstall": "node aio/scripts/version.js && ./node_modules/protractor/bin/webdriver-manager update --gecko=false && go get -u github.com/golangci/golangci-lint/cmd/golangci-lint && echo ${PATH} && echo ${GOPATH} && ls ${GOPATH}/bin && which golangci-lint"
```

Comment:
Great :) Remove echos and we can merge.

Reply:
please wait for a while. still something wrong.